financial-connections/src/main/java/com/stripe/android/financialconnections/domain/GoNext.kt

```diff
@@ -15,26 +15,31 @@ internal class GoNext @Inject constructor(
     private val logger: Logger
 ) {
 
-    operator fun invoke(nextPane: NextPane): NavigationCommand {
-        val nextPaneDirection = nextPane.toNavigationCommand()
+    operator fun invoke(
+        nextPane: NextPane,
+        args: Map<String, Any?> = emptyMap()
+    ): NavigationCommand {
+        val nextPaneDirection = nextPane.toNavigationCommand(args)
         logger.debug("Navigating to next pane: ${nextPaneDirection.destination}")
         navigationManager.navigate(nextPaneDirection)
         return nextPaneDirection
     }
 
     @Suppress("ComplexMethod")
-    private fun NextPane.toNavigationCommand(): NavigationCommand = when (this) {
+    private fun NextPane.toNavigationCommand(
+        args: Map<String, Any?>
+    ): NavigationCommand = when (this) {
         NextPane.INSTITUTION_PICKER -> NavigationDirections.institutionPicker
         NextPane.PARTNER_AUTH -> NavigationDirections.partnerAuth
         NextPane.CONSENT -> NavigationDirections.consent
         NextPane.ACCOUNT_PICKER -> NavigationDirections.accountPicker
         NextPane.SUCCESS -> NavigationDirections.success
+        NextPane.MANUAL_ENTRY -> NavigationDirections.manualEntry
+        NextPane.MANUAL_ENTRY_SUCCESS -> NavigationDirections.ManualEntrySuccess(args)
```

Comment:
override invoke on an object seems making this object mutable, could we instead have a more verbose creation method to convert args to navigationCommand in `ManualEntrySuccess` ?